harness-delegate-ng/values.yaml

```diff
@@ -138,3 +138,6 @@ custom_volumes:
 # minimum number of seconds for which a newly created Pod should be ready without any of its containers crashing, for it to be considered available.
 # This is set for improving stability during upgrade. It will tell kubernetes to wait at least this amount of seconds before removing the old pod after the new one becomes ready.
 minReadySeconds: 120
+
+ccm:
```

Comment:
1- Please provide some brief description about variable here.
2- Add Test cases in pr.
Rest LGTM

Reply:
tests and desc added, please review